diff -up kdenlive-0.7.8/src/colorcorrection/vectorscopegenerator.h.gcc kdenlive-0.7.8/src/colorcorrection/vectorscopegenerator.h
--- kdenlive-0.7.8/src/colorcorrection/vectorscopegenerator.h.gcc	2010-09-14 13:53:22.000000000 -0500
+++ kdenlive-0.7.8/src/colorcorrection/vectorscopegenerator.h	2011-04-15 07:38:06.156395786 -0500
@@ -12,8 +12,10 @@
 #define VECTORSCOPEGENERATOR_H
 
 #include <QObject>
+#include <QtGui/QImage>
 
-class QImage;
+// with added QtGui/QImage above, the forward declaration below probably is no longer needed -- Rex
+// class QImage;
 class QPoint;
 class QPointF;
 class QSize;
