--- /var/lib/mock/fedora-30-x86_64/root/builddir/build/BUILD/kdenlive-19.04.3/rttr.CMakeLists.txt	2019-07-08 16:41:42.000000000 +0100
+++ kdenlive-19.04.3/rttr.CMakeLists.txt	2019-09-22 03:21:31.065829482 +0100
@@ -1,7 +1,7 @@
 include(ExternalProject)
 ExternalProject_Add(rttr_project
     PREFIX rttr
-    URL https://github.com/rttrorg/rttr/archive/v0.9.6.tar.gz
+    SOURCE_DIR "rttr/"
     UPDATE_COMMAND ""
     CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=<INSTALL_DIR> -DCMAKE_INSTALL_LIBDIR=lib -DBUILD_EXAMPLES=OFF -DBUILD_DOCUMENTATION=OFF -DBUILD_UNIT_TESTS=OFF -DBUILD_PACKAGE=OFF -DBUILD_STATIC=ON
 )
